.travis.yml

```diff
@@ -2,6 +2,7 @@ language: julia
 os:
   - linux
 julia:
+  - 0.7
```

Comment:
I'd include a comment to replace `Memento.info(...)` with `info(...)` once we're dropping 0.7.

Reply:
Alternatively, you could do an
```julia
import Memento: info, warn
```
which will shadow the deprecated bindings from Base, with a comment saying to remove the `import` once 0.7 is dropped.

Reply:
Yeah, I like that solution better. I think that's what we used to do anyways.

Reply:
Oh shoot, I forgot to test this on 0.7. Seems like it doesn't work properly: https://travis-ci.org/invenia/Memento.jl/jobs/511627026